ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -134,7 +135,55 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
     public boolean shouldOverrideUrlLoading(WebView view, String url) {
         if (url.startsWith("http://") || url.startsWith("https://") ||
             url.startsWith("file://")) {
-          return false;
+              boolean shouldBlock = false;
```

Comment:
I'd say let's push this version as is and we can come up with a more general solution later.